ArduCopter/mode_guided.cpp

```diff
@@ -768,6 +779,163 @@ bool ModeGuided::limit_check()
     return false;
 }
 
+//loiter turns(circle) mode
+
+void ModeGuided::do_circle(const mavlink_command_long_t &packet)
+{
+    const Location circle_center = loc_from_param(packet);
+
+    // calculate radius
+    uint8_t circle_radius_m = radius_from_param(packet); // circle radius held in high byte of p1
```

Comment:
the comment don't match

Reply:
Sir, I have made the changes to the comment